api/src/main/java/org/pmiops/workbench/ras/RasLinkService.java

```diff
@@ -123,7 +124,7 @@ public RasLinkService(
   }
 
   /** Links RAS login.gov account with AoU account. */
-  public DbUser linkRasLoginGovAccount(String authCode, String redirectUrl) {
+  public DbUser linkRasAccount(String authCode, String redirectUrl) {
```

Comment:
This class has a fair amount of "login.gov" text which should be changed to RAS or made more generic.